### ISSUE TITLE
Connect external exchange by REST APIs?

### ISSUE THREAD

**Issue by ekanshdevops** [devcartel/pymt5#4]
Can I call REST APIs of an external exchange from PyMT5 that connects to devcartelgateway?

**Comment by wiwat-tharateeraparb** [comment]
Hi, PyMT5 doesn't have REST API. MT5 platform already have a REST API called WebAPI. This is might be what you looking for.

**Comment by ekanshdevops** [comment]
But the WebAPI can't be used to integrate MT5 with external exchange APIs? Also can I use PyMT5 to connect via FIX to external Exchange?

**Comment by wiwat-tharateeraparb** [comment]
What you need to do is to build a middleware or a connector like:

Exchange FIX <------> Application + PyMT5 <-------> DevCartelGateway64 + MT5

The gateway is built with MT5 Gateway API and must be installed at MT5 history server.

**Comment by ekanshdevops** [comment]
Yes Sure. Thanks! I just want to know how this PyMT5 application will connect to my application connecting to Exchange FIX? Can I have a documentation on PyMT5 in detail? Have sent you an invite as well on email to discuss further.

**Comment by TheSnowGuru** [comment]
please can check this , to connect to any metatrader
https://github.com/TheSnowGuru/PyTrader-python-mt4-mt5-trading-api-connector-drag-n-drop/
